index.html

```diff
@@ -0,0 +1,148 @@
+                        </li>
+                    </ul>
+                </div>
+                <div class="program-items">
+                    <ul>
+                        <li><i class="fa fa-car"></i></li>
+                        <li class="middle-text">Lecture</li>
+                        <li class="program-text">
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae, illo deserunt.
+                        </li>
+                    </ul>
+                </div>
+                <div class="program-items">
+                    <ul>
+                        <li><i class="fa fa-car"></i></li>
+                        <li class="middle-text">Lecture</li>
+                        <li class="program-text">
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae, illo deserunt.
+                        </li>
+                    </ul>
+                </div>
+                <div class="program-items">
+                    <ul>
+                        <li><i class="fa fa-car"></i></li>
+                        <li class="middle-text">Lecture</li>
+                        <li class="program-text">
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae, illo deserunt.
+                        </li>
+                    </ul>
+                </div>
+                <div class="program-items">
+                    <ul>
+                        <li><i class="fa fa-car"></i></li>
+                        <li class="middle-text">Lecture</li>
+                        <li class="program-text">
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae, illo deserunt.
+                        </li>
+                    </ul>
+                    <button class="btn">Join CC Global Summer 2015</button>
+                </div>
+            </div>
+            <div class="desktop-btn-container">
+                <a class="desktop-btn" href="">SEE THE WHOLE PROGRAM</a>
+            </div>
+        </section>
+        <section class="future-speakers-section">
+            <h2>Feature Speakers</h2>
+            <div class="niche">
+                <div></div>
+            </div>
+            <div class="future-speakers">
+                <div class="btn-container">
+                    <button class="load-btn">MORE</button>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer class="footer-section">
+        <h2>Partners</h2>
+        <div class="niche">
+            <div></div>
+        </div>
+        <div class="logo-container">
+            <img src="img/partner logo-4.png" alt=" Partner logo " />
+            <img src="img/partner-logo-5.png" alt="" />
+            <img src="img/partner-logo-3.png " alt="Partner logo " />
+            <img src="img/partner-logo-6.png " alt="Partner logo " />
+        </div>
```

Comment:
- [ ] I think it would be better if you separate the partner's section and the `footer`. And please make sure that what the  `footer` contains is a logo aligned to the left and text aligned to the right.